README.md

```diff
@@ -35,3 +35,21 @@ Bellow you can see the available services
 * POST `/log/error` (creates a log entry with error level) [message, stacktrace]
 
 Every services needs to send the apikey of the client on the http headers
+
+### Docker
+```
```

Comment:
Maybe explain in readme the purpose of this.
For local deployment or for pushing into aws, or why is needed, or for what would i want to create a docker.
Maybe specify the folder in wich to run this commands

Reply:
Not needed because locally we are not going to use docker.
Its only to remember for TC (When we have it I will remove it)